components/ui/tag/Select.vue

```diff
@@ -1,24 +1,25 @@
 <script setup lang="ts">
 const { selectedTags, availableTags, addTagWithEvent, removeTag } = useTags();
+
+const selectTag = ref<HTMLSelectElement | null>(null);
 </script>
 
 <template>
   <div class="flex m-0 gap-4 mt-4 py-2">
     <div v-for="myTag in selectedTags"
-      class="relative group border-2 shadow-sm font-semibold text-blue-500 bg-blue-600/10 rounded-lg px-1.5 py-1 inline-flex items-center justify-center"
-      :class="`border-blue-500`">
-      <span @click="() => removeTag(myTag)"
+      class="relative group border-2 shadow-sm font-semibold text-blue-500 bg-blue-600/10 rounded-lg px-1.5 py-1 inline-flex items-center justify-center border-blue-500"
+      :key="myTag"
+      >
+      <span @click="removeTag(myTag)"
         class="absolute text-gray-500 opacity-0 transition-all group-hover:opacity-100 hover:bg-gray-100 flex items-center justify-center -top-4 left-0 bg-white rounded-full h-6 w-6 border dark:bg-gray-700 dark:border-gray-600 dark:hover:bg-gray-800">
         <Icon name="tabler:x" />
       </span>
       {{ myTag }}
     </div>
-    <select :modelValue="null" @change="addTagWithEvent"
+    <select :modelValue="selectTag" @change="addTagWithEvent"
```

Comment:
I think here the model value should be removed because `addTagWithEvent` does everything we need.